CppSamples/EditData/SnapGeometryEdits/README.md

```diff
@@ -60,6 +60,8 @@ Snapping can be used during interactive edits that move existing vertices using
 
 Geometry guides are enabled by default when snapping is enabled. These allow for snapping to a point coinciding with, parallel to, perpendicular to or extending an existing geometry.
 
+On supported platforms haptic feedback on `SnapState.SnappedToFeature` and `SnapState.SnappedToGeometryGuide` is enabled by default when snapping is enabled. Custom haptic feedback can be configured by setting `SnapSettings.IsHapticFeedbackEnabled` to false and listening to `GeometryEditor.SnapChanged` events to provide specific feedback depending on the `SnapState`.
```

Comment:
let's update the mentioned methods to match what's in the C++ API (the method names are all uppercase)